core/templates/domain/email-dashboard/email-dashboard-backend-api.service.ts

```diff
@@ -55,7 +55,9 @@ export class EmailDashboardBackendApiService {
 
   fetchQueriesPage(
       pageSize: number, cursor: string): Promise<EmailDashboardQueryResults> {
-    let params: {'cursor'?: string; 'num_queries_to_fetch'?: string;} = {
+    // Here 'cursor' property is optional because it is present only if this
```

Comment:
I think you can always keep it but have it be null?

Also, try to define the whole "params" dict at once, rather than in stages.

Reply:
see comment below

Reply:
OK, please give a clearer comment (see below).

Reply:
done